api/controllers/helpers.go

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func responseHandle(w http.ResponseWriter, data interface{}, status int) {
+	w.Header().Add("Content-Type", "application/json")
+	w.WriteHeader(status)
+
+	if data == nil {
+		return
+	}
+	js, _ := json.Marshal(data)
+
+	w.Write(js)
```

Comment:
Never ignore errors; you should proper handle them.
```suggestion
	if data == nil {
		return
	}
	
	js, err := json.Marshal(data)
	if err != nil {
		// handle error
	}
	
	w.Write(js)
```